console-subscriber/src/aggregator/mod.rs

```diff
@@ -105,12 +105,12 @@ pub(crate) struct Flush {
     pub(crate) triggered: AtomicBool,
 }
 
-// An entity that at some point in time can be closed.
-// This generally refers to spans that have been closed
-// indicating that a task, async op or a resource is not
-// in use anymore
-pub(crate) trait Closable {
-    fn closed_at(&self) -> Option<SystemTime>;
+// An entity (e.g Task, Resource) that at some point in
+// time can be dropped. This generally refers to spans that
+// have been closed indicating that a task, async op or a
+// resource is not in use anymore
+pub(crate) trait Droppable {
+    fn dropped_at(&self) -> Option<SystemTime>;
 }
```

Comment:
What's the motivation for renaming this? I'm not a _big_ fan of having a trait called `Droppable`, since it seems like it could be easy to confuse with `Drop`. `Droppable` doesn't really indicate that something _can_ be dropped, since _everything_ can be dropped...instead, it indicates that something _has a dropped timestamp_. Maybe we should call this something like `DroppedTime` or `DroppedAt`?

Reply:
The main motivation was that I renamed the proto field (where it already existed) from `closed_at` to `dropped_at`. But yes, that all make sense. I will rename it.